### PR TITLE
[Network] Support --tcp-port-behavior while configuring a TCP configuration of a Connection Monitor V2

### PR DESCRIPTION
**Pull request by haroldrandom**
**Description**  

Close https://github.com/Azure/azure-cli/issues/14527

**Testing Guide**  

**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Network

File: src/azure-cli/azure/cli/command_modules/network/custom.py [hunk 1]
```diff
@@ -4319,7 +4324,8 @@ def _create_nw_connection_monitor_v2_test_configuration(cmd,
         ConnectionMonitorTcpConfiguration = cmd.get_models('ConnectionMonitorTcpConfiguration')
         tcp_config = ConnectionMonitorTcpConfiguration(
             port=tcp_port,
-            tcp_disable_trace_route=tcp_disable_trace_route
+            destination_port_behavior=tcp_port_behavior,
+            disable_trace_route=tcp_disable_trace_route
```
Comment (MyronFanQiu) [inline src/azure-cli/azure/cli/command_modules/network/custom.py:4328]: Does if change the name of the property?

**Review by MyronFanQiu — COMMENTED** [review]
one test?

File: src/azure-cli/azure/cli/command_modules/network/custom.py (hunk 1 above)
Comment (haroldrandom) [reply src/azure-cli/azure/cli/command_modules/network/custom.py:4328]: No. It's typo last time I did here

**Review by MyronFanQiu — APPROVED** [review]

**Pull request merged**